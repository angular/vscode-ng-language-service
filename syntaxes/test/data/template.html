<!-- Interpolation test -->
<div>{{ call(1 + 2 + 3) }}</div>

<!-- Property binding test -->
<div [ ngStyle ]="{'max-width.px': i * 2 + 5}"></div>
