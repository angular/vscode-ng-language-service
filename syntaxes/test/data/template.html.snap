><!-- Interpolation test -->
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^ template.ng
><div>{{ call(1 + 2 + 3) }}</div>
#^^^^^ template.ng
#     ^^ template.ng punctuation.definition.block.ts
#       ^^^^^^^^^^^^^^^^^ template.ng source.js
#                        ^^ template.ng punctuation.definition.block.ts
#                          ^^^^^^^ template.ng
>
><!-- Property binding test -->
#^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ template.ng
><div [ ngStyle ]="{'max-width.px': i * 2 + 5}"></div>
#^^^^^ template.ng
#     ^^^^^^^^^^^ template.ng entity.other.attribute-name.html
#                ^ template.ng punctuation.separator.key-value.html
#                 ^ template.ng punctuation.definition.string.begin.html string.quoted.html
#                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ template.ng source.js
#                                             ^ template.ng punctuation.definition.string.end.html string.quoted.html
#                                              ^^^^^^^^ template.ng
>